model/godleyTable.h

```diff
@@ -106,6 +106,14 @@ namespace minsky
        sets if assetClass present, otherwise gets
     */
     string assetClass(ecolab::TCL_args args);
+    
+    // returns the asset class of column col. For ticket 1072.
+    AssetClass getAssetClass(unsigned col) const {
```

Comment:
This does duplicate the method above. (assetClass).

Reply:
Sorry, I was thrown off by its type (string) and the appearance of its argument. So I went back to getAssetClass(). But I've now replaced getAssetClass() by assetClass(). Thanks.

Reply:
getAssetClass now dead code? Of would you prefer to rename it to assetClass, and delete the existing setter/getter on line 108? I believe the setter is not used anywhere in any TCL file (maybe it was once).

Reply:
Sorry, I forgot to commit this change. I removed it from my develop folder but not the one from which I do commits, so it didn't show up for the commit as the other changes did. I will remove it.